src/ResourceManagement/AnalysisServices/AnalysisServices.Tests/Helpers/AnalysisServicesTestUtilities.cs

```diff
@@ -0,0 +1,215 @@
+                                'name':'{4}'
+                            }},
+                            'tags':{5},
+                            'properties':{{
+                                'provisioningState':'{6}',
+                                'serverFullName':'asazure://wcus.asazure-int.windows.net/{2}',
+                                'asAdministrators':{{
+                                'members':{7}
+                                }}
+                            }}
+                            }}";
+            
+            var tags = JsonConvert.SerializeObject(DefaultTags, new KeyValuePairConverter());
+            var admins = JsonConvert.SerializeObject(DefaultAdministrators);
+            return string.Format(
+                responseFormat,
+                testSubscription,
+                DefaultResourceGroup,
+                DefaultServerName,
+                DefaultLocation,
+                DefaultSku.Name,
+                tags,
+                provisioningState,
+                admins);
+        }
+
+        public static ResourceManagementClient GetResourceManagementClient(MockContext context, RecordedDelegatingHandler handler)
+        {
+            if (IsTestTenant)
+            {
+                return null;
+            }
+            else
+            {
+                handler.IsPassThrough = true;
+                ResourceManagementClient resourcesClient = context.GetServiceClient<ResourceManagementClient>(handlers: handler);
+                return resourcesClient;
+            }
+        }
+
+        public static AnalysisServicesResource GetDefaultAnalysisServicesResource()
+        {
+            AnalysisServicesResource defaultServer = new AnalysisServicesResource
+            {
+                Location = DefaultLocation,
+                Tags = DefaultTags,
+                Sku = DefaultSku,
+                AsAdministrators = new ServerAdministrators(DefaultAdministrators)
+            };
+
+            return defaultServer;
+        }
+
+        public static AnalysisServicesClient GetAnalysisServicesClient(RecordedDelegatingHandler handler)
+        {
+            handler.IsPassThrough = false;
+            var client = new AnalysisServicesClient(new TokenCredentials("xyz"), handler);
+            client.SubscriptionId = "00000000-0000-0000-0000-000000000000";
+            if (HttpMockServer.Mode != HttpRecorderMode.Record)
+            {
+                client.LongRunningOperationRetryTimeout = 0;
+            }
+            return client;
+        }
+
+        public static AnalysisServicesClient GetAnalysisServicesClient(MockContext context, RecordedDelegatingHandler handler)
+        {
+            AnalysisServicesClient analysisServicesClient;
+            if (IsTestTenant)
+            {
+                analysisServicesClient = new AnalysisServicesClient(new TokenCredentials("xyz"), GetHandler());
+                analysisServicesClient.SubscriptionId = testSubscription;
```

Comment:
>testSubscription [](start = 56, length = 16)

Any reason you are not using TestEnvironment and connection string to set all your credentials and subscription information?